pihole_adlist_tool

```diff
@@ -350,12 +351,22 @@ if [ "$UNIQUE" -eq 1 ];
         echo -e "  [i]  UNIQUE: Not shown"
 fi
 
+# do not RegEx when running in docker container:
+if [ "$PIHOLE_DOCKER" = 1 ]; then
+  REGEX_MODE=0
+else
+  REGEX_MODE=1
+fi
 # print if RegEx should be analysed as well
 if [ "$REGEX_MODE" -eq 1 ];
     then
         echo -e "  [i]  REGEX_MODE: Enabled"
     else
+      if [ "$PIHOLE_DOCKER" = 0 ]; then
         echo -e "  [i]  REGEX_MODE: Disabled"
+      else
+        echo -e "  [i]  REGEX_MODE: Disabled due to running Pi-hole in Docker Container"
```

Comment:
You need to move the disabling logic after printing the output. Otherwise you can't discriminate on docker if users did not choose to  run RegEx or if it was disabled by us.

Reply:
That's right, too.